tests/unit/assessment/test_clusters.py

```diff
@@ -265,3 +266,37 @@ def test_try_fetch():
     assert result_set[0].cluster_id == "000"
     assert result_set[0].success == 1
     assert result_set[0].failures == "123"
+
+
+def test_no_isolation_clusters():
+    sample_clusters = [
+        ClusterDetails(
+            cluster_name="No isolation shared",
+            spark_version="12.3.x-cpu-ml-scala2.12",
+            data_security_mode=DataSecurityMode.NONE,
+        )
+    ]
+    ws = workspace_client_mock()
```

Comment:
can we load the cluster config from a json file, similar to other cluster unit test?

Reply:
The test is relatively simple and doesn't really need to have a JSON file similar to the other tests that carries more complex configurations and cases